mailer.js

```diff
@@ -209,15 +209,26 @@ module.exports = function (log) {
     this.mailer.sendMail(
       emailConfig,
       function (err, status) {
-        log.trace(
-          {
-            op: 'mailer.send.1',
-            err: err && err.message,
-            status: status && status.message,
-            id: status && status.messageId
-          }
-        )
-        return err ? d.reject(err) : d.resolve(status)
+        if (err) {
+          log.error(
+            {
+              op: 'mailer.send.1',
+              err: err && err.message,
+              status: status && status.message,
+              id: status && status.messageId
+            }
+          )
+          d.reject(err)
+        } else {
+          log.trace(
```

Comment:
Actually, I'd prefer to be verbose about the success of the actual send (so `info`). (Note: `trace` is effectively `/dev/null` in production).